README.md

```diff
@@ -108,8 +88,5 @@ This will build the container image and spit out a tag at the end, now we'll nee
 ```
 docker push quay.io/<username>/aws-account-shredder:<tag>
 ```
-You'll now need to update `deploy/deployment.yaml` to point at the image you just built:
-```
-image: quay.io/<username>/aws-account-shredder:<image tag>
-```
-Now that you have the your image pushed up and your deployment updated, you can simply apply the deployment to roll out your local test branch.
+
+Now that you have your image pushed up, you can simply apply the deployment again using `make deploy`
```

Comment:
does the user need to set a IMG env variable or something like that so the make will use the correct image?

Reply:
```suggestion
Now that you have your image pushed up, you can apply the deployment again using `make deploy`
```
words like simply or easy should be banned from all documentation. What's simple for you, can be complicated for someone else ;)

Reply:
> does the user need to set a IMG env variable or something like that

Setting the image repository is documented a couple of lines above the one i've changed, so it did not appear in the diff:
```
You'll need to edit the `IMAGE_REPOSITORY` field in `project.mk` to point at your `aws-account-shredder` repo 
```

Reply:
Alternatively, ( and probably better ), one can override the variable on the commandline or with an env var:
```
make docker-build IMAGE_REPOSITORY=maschulz
```
and
```
export IMAGE_REPOSITORY=maschulz
make docker-build
```
achieve exactly the same. Do you have a preference for what should be documented as the "default" way?

Reply:
I added a line to the readme to clarify this a bit.